.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+**Point to the API documentation/Plugin/feature? Please describe.**
```

Comment:
Please don't use slash in text. Its ambiguous at best. Style guides say not to use "please".
s|Point to the API documentation/Plugin/feature? Please describe.|Point to the API documentation, Plugin, or feature and describe it.|